core/src/main/java/org/ldaptive/io/FileResourceLoader.java

```diff
@@ -0,0 +1,38 @@
+/* See LICENSE for licensing and NOTICE for copyright. */
+package org.ldaptive.io;
+
+import java.io.File;
+import java.io.FileInputStream;
+import java.io.IOException;
+import java.io.InputStream;
+
+/**
+ * Creates an {@link InputStream} from a string that is prefixed with 'file:'. See {@link
+ * FileInputStream#FileInputStream(File)}.
+ *
+ * @author  Middleware Services
+ */
+public class FileResourceLoader implements ResourceLoader
```

Comment:
I wonder if we should simply take the opportunity to broaden scope to load anything that `URL#openConnection()` can handle, which would cover the existing filesystem use case as well as the HTTP(S) use case. The component name would be `URLResourceLoader` in that case.

Reply:
Note that this patch gets us further to that goal with the URLResourceLoader, but does not wire it by default.

Reply:
Yes, I saw that and don't have concerns at this time.